challenge_10/rack_response.rb

```diff
@@ -39,37 +33,34 @@ def set_up_blogs_store(store_path)
   # Build Rack request
   request = Rack::Request.new(environment)
 
-  headers = {}
-  body = []
+  # Build Rack response
+  response = Rack::Response.new
 
   if request.get? && request.path == '/show-data'
-    body << '<ul>'
+   response.write '<ul>'
```

Comment:
The indentation went a little funny again, but I can live with it 😂

Reply:
Bahahaha let me fix this